packages/react-components/react-table/src/hooks/useInteractiveKeyboardResizing.ts

```diff
@@ -0,0 +1,95 @@
+import * as React from 'react';
+import { ArrowLeft, ArrowRight, Enter, Escape, Shift, Space } from '@fluentui/keyboard-keys';
+import { useEventCallback } from '@fluentui/react-utilities';
+import { ColumnResizeState, TableColumnId } from './types';
+import { useFluent_unstable as useFluent } from '@fluentui/react-shared-contexts';
+
+const STEP = 20;
+const PRECISION_MODIFIER = Shift;
+const PRECISION_FACTOR = 1 / 4;
+
+export function useInteractiveKeyboardResizing(columnResizeState: ColumnResizeState) {
```

Comment:
```suggestion
export function useKeyboardResizing(columnResizeState: ColumnResizeState) {
```

is interactive necessary here?